client/my-sites/post-type-list/index.jsx

```diff
@@ -46,6 +46,7 @@ class PostTypeList extends Component {
 		super( ...arguments );
 
 		this.renderPostRow = this.renderPostRow.bind( this );
+		this.cellRendererWrapper = this.cellRendererWrapper.bind( this );
```

Comment:
Ah, in retrospect, you're right. Case in point of why it can be better to err on the safe side 😆